x/staking/keeper/historical_info_test.go

```diff
@@ -76,15 +76,20 @@ func TestTrackHistoricalInfo(t *testing.T) {
 	require.True(t, found)
 	require.Equal(t, hi5, recv)
 
-	// Set last validators in keeper
+	// Set bonded validators in keeper
 	val1 := teststaking.NewValidator(t, addrVals[2], PKs[2])
+	val1.Status = types.Bonded
+	val1.Tokens = sdk.NewInt(100)
 	app.StakingKeeper.SetValidator(ctx, val1)
-	app.StakingKeeper.SetLastValidatorPower(ctx, val1.GetOperator(), 10)
+	app.StakingKeeper.SetValidatorByPowerIndex(ctx, val1)
 	val2 := teststaking.NewValidator(t, addrVals[3], PKs[3])
-	vals := []types.Validator{val1, val2}
-	sort.Sort(types.Validators(vals))
+	val2.Status = types.Bonded
+	val2.Tokens = sdk.NewInt(80)
 	app.StakingKeeper.SetValidator(ctx, val2)
-	app.StakingKeeper.SetLastValidatorPower(ctx, val2.GetOperator(), 8)
+	app.StakingKeeper.SetValidatorByPowerIndex(ctx, val2)
+
+	vals := []types.Validator{val1, val2}
```

Comment:
I think this is already sorted, want to switch the ordering?

Reply:
It is already sorted, but it is sorted first by power and then by operator address. I want to sort first by power and then by consensus address (sadly doesn't already exist in SDK)

Reply:
Yea, but why not have the `Sort` function do the sorting? Or is the `Sort` just a sanity check?

Reply:
Ohhh I see your question. Yea I can switch them.